test/Table_Tests/src/Util.enso

```diff
@@ -16,7 +17,7 @@ Column.Column.should_equal expected =
     self.length.should_equal expected.length
     self.to_vector.should_equal expected.to_vector
 
-normalize_lines string line_separator=System.default_line_separator newline_at_end=True =
+normalize_lines string line_separator='\n' newline_at_end=True =
```

Comment:
That should probably remain as a constant somewhere?

Reply:
```suggestion
normalize_lines string line_separator=Line_Ending_Style.Unix.to_text newline_at_end=True =
```

Does that look ok?